stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -257,12 +257,22 @@ public class MessageInputView : ConstraintLayout {
                 binding.messageInputFieldView.binding.messageEditText.focusAndShowKeyboard()
             }
 
+            is InputMode.EditModeratedMessage -> {
```

Comment:
Let's try to get rid of this new input mode type as it's the same as Edit, except for one function (onEditModeratedMessage vs onEdit). I wouldn't make a distinction between the two, as they both do the same